### ISSUE TITLE
Is JTable still being developed?

### ISSUE THREAD

**Issue by DeveloperMCD** [volosoft/jtable#1469]
It looks like the project hasn't seen activity (aside from new "issues" being created) for the past 6 months. Please tell me I'm wrong! :) I love this plugin.

**Comment by AngelusWeb** [comment]
How about now? It's July 2015, and not much sign of updates.

**Comment by hikalkan** [comment]
Hi,

jTable project is not dropped. I love it and using in our products. We can fix if you find a bug. Still I've no much time to add new features. But there is a good news, I will have more time in near future and still want much to improve jTable.

**Comment by sheryever** [comment]
@hikalkan nice to listen :+1:

I am also using jTable in my all projects and i want to contribute. If you have any working then you can assign that.

**Comment by hikalkan** [comment]
I'll have more time, since: http://halilibrahimkalkan.com/why-i-left-my-job/

**Comment by Maktos** [comment]
I was excited to hear that you were going to spend more time on JTable again, in this post of July 21st. But now it's October 12th, and I don't see a lot of activity here. Am I missing something? Did you change your mind and get another full-time job? Just wondering.

**Comment by hikalkan** [comment]
No, I did not get any job. But as you guess it's hard to establish a company and I'm busy on other projects. I will have time in the future.
Actually I want to re-write jTable using Typescript and make it more extendable. But unfortunately I can not declare an exact date for these works.
I love and use jTable for my projects.

**Comment by sheryever** [comment]
@hikalkan :+1: I am also planing to rewrite the client side section of my application architecture to TypeScript. I have also put my db orm on github ([SimpleAccess](https://github.com/sheryever/simple-access-orm/tree/Development)).

**Comment by sheryever** [comment]
@hikalkan Is there any update regarding new version or further development. My office team is forcing to drop jtable.

**Comment by sheryever** [comment]
@hikalkan if you need any help then you can assign the task to me, I will create the pull request for  each feature

**Comment by hebol** [comment]
Any ideas on releasing a new version?

**Comment by hikalkan** [comment]
Hi,

Sorry for not have time for it yet. But I scheduled a few days to review changes, pull requests and publish a new release, in 3-4 weeks.

**Comment by EvgeniyVasyliev** [comment]
Hi, Hikalkan!

I wanted to check if your plans to publish a new release are still valid? Thanks.

**Comment by hikalkan** [comment]
Yes, but could not have time yet, sorry.
